ui/packages/catalog-ui-search/src/main/webapp/component/query-add/query-add.view.js

```diff
@@ -224,13 +249,46 @@ module.exports = Marionette.LayoutView.extend({
     },
     saveTemplateToBackend: function() {
         let loadingView = new LoadingView();
+        let _this = this;
+        let _user = user;
         $.ajax({
             url: '/search/catalog/internal/forms/query',
             data: JSON.stringify(this.getQueryAsQueryTemplate()),
             method: 'PUT',
             contentType: 'application/json',
             customErrorHandling: true
         })
+        .done((data, textStatus, jqxhr) => {
```

Comment:
❓ Is the purpose of this entire done callback to set the current template to whatever the user just saved?

Reply:
Even on the re-query it would not update a search form it already knew about so it was an update problem. This helps make created search forms immediately available without a refresh.